server/codelogs/codelog_handler.coffee

```diff
@@ -0,0 +1,24 @@
+CodeLog = require './CodeLog'
+Handler = require '../commons/Handler'
+mongoose = require 'mongoose'
+
+CodeLogHandler = class CodeLogHandler extends Handler
+  modelClass: CodeLog
+  makeNewInstance: (req) ->
+    codelog = super(req)
+    codelog.id = req.id
+    codelog.set 'sessionID', req.body.sessionID
+    codelog.set 'level', req.body.level
+    codelog.set 'levelSlug', req.body.levelSlug
+    codelog.set 'userID', req.body.userID
+    codelog.set 'userName', req.body.userName
+    codelog.set 'log', req.body.log
+    codelog.set 'date', new Date().toISOString()
+    codelog.set 'id', req.body.id
+
+    return codelog
+
+  post: (req, res) ->
+    super(req, res)
+
+module.exports = new CodeLogHandler()
```

Comment:
Handlers are deprecated. Instead, add routes to [`server/routes/index.coffee`](https://github.com/codecombat/codecombat/blob/master/server/routes/index.coffee), using middleware. You just need to GET a collection of them and POST new ones? Do something like [Article's routes here](https://github.com/codecombat/codecombat/blob/master/server/routes/index.coffee#L29-L30). You'll need to add `editableProperties` to the server CodeLog model, like [here](https://github.com/codecombat/codecombat/blob/master/server/models/Article.coffee#L39-L40). And there should be server tests like for [Article GET](https://github.com/codecombat/codecombat/blob/master/spec/server/functional/article.spec.coffee#L10-L27) and [Article POST](https://github.com/codecombat/codecombat/blob/master/spec/server/functional/article.spec.coffee#L121-L139). If you have any questions about this server stuff, let me know!